packages/bundler/src/modules/BundleManager.ts

```diff
@@ -65,7 +74,8 @@ export class BundleManager {
         debug('sendNextBundle - no bundle to send')
       } else {
         const beneficiary = await this._selectBeneficiary()
-        const ret = await this.sendBundle(bundle, beneficiary, storageMap)
```

Comment:
remove: this method is used in 4337 mode only.

Reply:
Remove the `STOPSHIP` comment, right? If so, done.